src/test/java/com/github/fishio/achievements/TestAchievement.java

```diff
@@ -0,0 +1,67 @@
+package com.github.fishio.achievements;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+/**
+ * This test class tests the Achievement class.
+ *
+ */
+public class TestAchievement {
+	
+	/**
+	 * Tests the {@link Achievement#Achievement(String, int)} constructor.
+	 */
+	@Test
+	public void testAchievement1() {
+		Achievement achievetest1 = new Achievement("Testachievement", 3);
+	}
+	
+	/**
+	 * Tests the {@link Achievement#Achievement(String)} constructor.
+	 */
+	@Test
+	public void testAchievement2() {
+		Achievement achievetest2 = new Achievement("Testachievement");
+	}
+	
```

Comment:
When testing if the constructor works you should do some kind of check that the constructor initialized correctly. 

https://blackboard.tudelft.nl/bbcswebdav/pid-2391002-dt-content-rid-7957602_2/courses/34547-141501/TI1206-Q1-15-ExtraTesten.pdf 

<testing